src/data/setoid/partition.lean

```diff
@@ -237,6 +244,26 @@ _ (subtype (@is_partition α)) _ (partial_order.to_preorder _) $ partition.order
 
 end partition
 
+section finpartition
+
+open_locale classical
+
+/-- A finite setoid partition furnishes a finpartition -/
+def _root_.is_partition.finpartition {c : finset (set α)}
```

Comment:
Sorry, I got confused here: the original name was correct, of
```suggestion
def is_partition.finpartition {c : finset (set α)}
```
which results in the name `_root_.setoid.is_partition`, which is what we need for `hc.finpartition` to work as dot notation.

Reply:
OK, reverted.